### PR TITLE
Homework 01 Alex Lahood

### PR DESCRIPTION
**Pull request by alexlahood**
---
name: PR template for CFSS
about: Used for students submitting a pull request for a homework assignment
title: Description of my submission
labels: ''
assignees: ''

---

Before submitting your pull request, have you:

* Included all relevant files in your fork (e.g. R Markdown files, rendered Markdown files, images/graphs)?
* Added or updated a relevant `README.md` file describing the purpose of your repository, required packages, location of relevant files, etc.?

Be sure to include your name somewhere in the title or description of your pull request, as well as reflect on what was hard/easy, problems you solved, helpful tutorials you read, etc.

### CODE REVIEW

**Review by pdiazm — COMMENTED** [review]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |           |      x        |            |
| **Coding strategy**         |           |       x      |            |
| **Presentation**            |     x      |              |            |
| **Achievement, creativity** |    x       |              |            |
| **Ease of access**          |      x     |              |            |

Remarks:

* Overall, your updates look very good and I like the light tone of your content. 
* I graded the coding style and strategy as satisfactory because the link didn't work. I believe you forgot to include the full URL.
* Example: [link](https://www.ted.com/#/)

File: README.md [hunk 1]
```diff
@@ -1 +1,27 @@
-# hw01
\ No newline at end of file
+---
+output:
```
Comment (pthelen124) [inline README.md:2]: README files are rendered automatically to GitHub; so you don't need to add this form of YAML metadata. These will be necessary for `.Rmd` files for future homeworks, but not for the READMEs.

File: README.md [hunk 2]
```diff
@@ -1 +1,27 @@
...
+  + *Get Out*
+  + *The Incredibles II*
+
+* The beach
+* All types of music
+
+
+Thank you for coming to my TED Talk. 
+
+[link](www.TED.com)
```
Comment (pthelen124) [inline README.md:26]: Make sure that the link you're adding is **exactly** as the homepage (in this case) that you're trying to link to. In this case, attempting to reach the link: `www.TED.com` automatically redirects to `https://www.ted.com/#/`. Without the correct text, GitHub tried to route to your link _through_ your repository, so it returned an error.

**Review by pthelen124 — COMMENTED** [review]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |           |   x           |            |
| **Coding strategy**         |           |              |   x         |
| **Presentation**            |           |    x          |            |
| **Achievement, creativity** |           |    x          |            |
| **Ease of access**          |   x        |              |            |

* You did a good job of leaving blank lines between headers and plain text. This practice will come in handy for later assignments. 
* Nice work on this assignment!

**Comment by kadenhan** [comment]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |           |x              |            |
| **Coding strategy**         |        x   |              |            |
| **Presentation**            |        x   |              |            |
| **Achievement, creativity** |       x    |              |            |
| **Ease of access**          |        x   |              |            |

Remarks:

* As said earlier, the link doesn't quite work
* I like the different format for the sublists, different from how I did it

**Pull request closed without merging**